common/src/cache.rs

```diff
@@ -64,28 +64,35 @@ pub fn load_animation_frames(
     Ok(None)
 }
 
-pub fn get_previous_image_path(output_name: &str) -> io::Result<String> {
+pub fn get_previous_image_path(output_name: &str) -> io::Result<(String, String)> {
     let mut filepath = cache_dir()?;
     clean_previous_verions(&filepath);
 
     filepath.push(output_name);
     if !filepath.is_file() {
-        return Ok("".to_string());
+        return Ok(("Lanczos3".to_string(), "".to_string()));
```

Comment:
I don't really get what this gives us. Is it just to improve the error messages?

Reply:
Yes, and it makes it easy to force swww-daemon ignore missing cache files.